.github/workflows/ci.yaml

```diff
@@ -77,3 +77,109 @@ jobs:
           file: ./dockerfiles/components/chainhook-node.dockerfile
           # Only push if (there's a new release on main branch, or if building a non-main branch) and (Only run on non-PR events or only PRs that aren't from forks)
           push: ${{ (github.ref != 'refs/heads/master' || steps.semantic.outputs.new_release_version != '') && (github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository) }}
+
+  devploy-dev:
+    runs-on: ubuntu-latest
+    needs:
+      - build-publish
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository
+    env:
+      DEPLOY_ENV: dev
+    environment:
+      name: Development
```

Comment:
These 3 envs need to have required reviewers configured, otherwise it'll just deploy to them without an approval. [You can configure that here.](https://github.com/hirosystems/chainhook/settings/environments)

Reply:
opps yeah, I added the reviewers but forgot to save the changes